Business Rules/Custom Scripted Field for Calculating Resolution Time/Script.js

```diff
@@ -0,0 +1,10 @@
+// Business Rule: Calculate Resolution Time
+// Trigger: After Update
+// Table: incident
+
+if (current.state == 'Closed' && current.resolved_at) {
```

Comment:
This will not work as state's value will not be `Closed`. Further, use `.getValue('field')`